waku/waku_archive_legacy/driver/postgres_driver/postgres_healthcheck.nim

```diff
@@ -0,0 +1,38 @@
+{.push raises: [].}
+
+import chronos, chronicles, results
+import ../../../common/databases/db_postgres, ../../../common/error_handling
+
+## Simple query to validate that the postgres is working and attending requests
+const HealthCheckQuery = "SELECT version();"
+const CheckConnectivityInterval = 60.seconds
+const MaxNumTrials = 20
+const TrialInterval = 1.seconds
+
+proc checkConnectivity*(
+    connPool: PgAsyncPool, onFatalErrorAction: OnFatalErrorHandler
+) {.async.} =
+  while true:
+    (await connPool.pgQuery(HealthCheckQuery)).isOkOr:
+      ## The connection failed once. Let's try reconnecting for a while.
+      ## Notice that the 'exec' proc tries to establish a new connection.
```

Comment:
I don't understand this comment. What `exec`?

Reply:
> I don't understand this comment. What `exec`?

Good catch! I updated the comment. Thanks!